.expeditor/end_to_end.pipeline.yml

```diff
@@ -29,6 +29,30 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
 
+  - label: "[:linux: hup-does-not-abandon-services]"
+    command:
+      - .expeditor/scripts/setup_environment.sh DEV
+      - hab pkg install --binlink core/expect/5.45.4/20190115014137
```

Comment:
The reason the fully-qualified ident is needed here is because the environment contains `HAB_BLDR_CHANNEL=DEV` (see line 13 above), and there isn't a `core/expect` in that channel.

Since the exact release of `expect` isn't really important here, we could simplify this to 

```sh
hab pkg install --binlink --channel=stable core/expect/5.45.4
```
or, simpler still, to

```sh
hab pkg install --binlink --channel=stable core/expect
```

Reply:
What's the advantage? I don't think `expect` is likely to change in a way that's relevant to this test. Pinning it has the advantage to eliminating a potential difference when running this in different environments.

Reply:
It's just that when I see a pin, I immediately wonder what the significance of that specific release is.

Reply:
Fair enough. I'll try changing it.